yoti_python_sdk/docs/__init__.py

```diff
@@ -0,0 +1,19 @@
+from .session.create.check.document_authenticity import (
```

Comment:
Should these files live under `yoti_python_sdk/doc_scan/...`? (rather than `docs`)

Reply:
Changed in 9077b7a